.github/workflows/ci.yml

```diff
@@ -91,6 +91,42 @@ jobs:
           start: npm run dev:standalone
           wait-on: "http://localhost:3000/search"
 
+  api-build:
+            name: Build API
+            runs-on: ubuntu-latest
+
+            env:
+              api-directory: ./api
+
+            strategy:
+                matrix:
+                  dotnet-version: [ '9.0.x' ]
+
+            defaults:
+              run:
+                working-directory: ${{ env.api-directory }}
+
+            steps:
+              - name: Checkout source
+                uses: actions/checkout@v4
+
+              - name: Setup dotnet ${{ matrix.dotnet-version }}
+                uses: actions/setup-dotnet@v3
+                with:
+                  dotnet-version: ${{ matrix.dotnet-version }}
+
+              - name: Display dotnet version
+                run: dotnet --version
+
+              - name: Restore Packages
+                run: dotnet restore
+
+              - name: Build API
+                run: dotnet build
+
+              - name: Run Unit Tests
+                run: dotnet test
```

Comment:
This indentation looks off. I don't think the contents of the `api-build` job need to be indented so far, do they?

Reply:
No they don't - reformatted as suggested.